packages/turbo-ignore/README.md

```diff
@@ -8,4 +8,48 @@ $ npx turbo-ignore
 
 This uses `turbo` to automatically determine if the current app has new changes that need to be deployed.
 
-For more information about Turborepo, [visit turbo.build](https://turbo.build) and follow us on Twitter ([@turborepo](https://twitter.com/turborepo))!
+## Usage
+
+Use `npx turbo-ignore --help` to see list of options:
+
+```sh
+turbo-ignore
+
+Automatically ignore builds that have no changes
+
+Usage:
+  $ npx turbo-ignore [<workspace>] [flags...]
+
+If <workspace> is not provided, it will be inferred from the "name"
+field of the "package.json" located at the current working directory.
+
+Flags:
+  --filter-fallback   If no previously deployed SHA is available to compare with,
+                      fallback to comparing against the previous commit [default: true]
+  --help, -h          Show this help message
+  --version, -v       Show the version of this script
+```
+
+## How it Works
+
+`turbo-ignore` determines if a build should continue by analyzing the dependency graph of the given workspace.
+
+The _given workspace_ is determined by reading the "name" field in the "package.json" file located at the current working directory, or by passing in a workspace name as the first argument to `turbo-ignore`.
+
+Next, it used `turbo` to determine if the given workspace, _or any dependencies of the workspace_, have changed since the previous commit.
```

Comment:
```suggestion
Next, it uses `turbo run build --dry` to determine if the given workspace, _or any dependencies of the workspace_, have changed since the previous commit.
```